engine/faq.md

```diff
@@ -47,23 +47,23 @@ lightweight and perfect as a unit of software delivery.
 
 ### What does Docker technology add to just plain LXC?
 
-Docker technology is not a replacement for LXC. "LXC" refers to capabilities of
+Docker technology is not a replacement for [LXC](https://linuxcontainers.org/). "LXC" refers to capabilities of
 the Linux kernel (specifically namespaces and control groups) which allow
 sandboxing processes from one another, and controlling their resource
 allocations. On top of this low-level foundation of kernel features, Docker
 offers a high-level tool with several powerful functionalities:
 
  - *Portable deployment across machines.* Docker defines a format for bundling
- an application and all its dependencies into a single object which can be
- transferred to any Docker-enabled machine, and executed there with the
+ an application and all its dependencies into a single object (called a container) which can be
```

Comment:
Consider "into a single object called a container. This container can be transferred to any Docker-enabled machine, where it can be executed with the guarantee that...